src/admin/components/App.tsx

```diff
@@ -50,29 +42,29 @@ const App = () => {
         list={modules.user.List}
         show={modules.user.Show}
         edit={modules.user.Edit}
-        icon={UserIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.USERS} />}
       />
       <Resource
         name={modules.organisation.ResourceName}
         list={modules.organisation.List}
         show={modules.organisation.Show}
         edit={modules.organisation.Edit}
-        icon={BusinessIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.ORGANISATIONS} />}
       />
       <Resource
         name={modules.pitch.ResourceName}
         list={modules.pitch.List}
         show={modules.pitch.Show}
         edit={modules.pitch.Edit}
-        icon={ForestIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.PITCHES} />}
       />
       <Resource
         name={modules.fundingProgramme.ResourceName}
         list={modules.fundingProgramme.List}
         edit={modules.fundingProgramme.Edit}
         show={modules.fundingProgramme.Show}
         create={modules.fundingProgramme.Create}
-        icon={AttachMoneyIcon}
+        icon={() => <Icon className="h-8 w-8" name={IconNames.FUJDING_PROGRAMMES} />}
```

Comment:
This icon name is odd - I don't see it defined in this PR; what's it meant to signify?

Reply:
Thank you for pointing that out! The icon name was indeed incorrect. I have updated it to the correct name, IconNames.FUNDING_PROGRAMMES, which is defined and signifies the funding programmes icon.